index.html

```diff
@@ -9,30 +9,38 @@
     <title>Portfolio Project</title>
 </head>
 <body>
+    <header>
     <nav class="nav-bar">
+        <a href="#">
         <h1 class="title">Welcome!</h1>

```

Comment:
- [ ] Please know that as the previous reviewer mentioned, the `hx` tags should be used to organize information and orient users the logo isn't one of those, kindly change that. 😁
- [ ] Also when you added the `a` tag the styling changed for the logo, kindly fix that.

Currently it's like this in your website:
![image](https://user-images.githubusercontent.com/20567503/176574986-fe3f5d8f-6f1c-4087-954c-58c603d6861a.png)
But according to the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468) it should be like this:
![image](https://user-images.githubusercontent.com/20567503/176575021-6c1f4822-74de-4094-9e4b-310a6418a524.png)

Reply:
I removed the <h1> tag, hadn't understood that previously. I didn't know which tag to use for the logo-text, so I used the <p> tag, hope that is correct.
Also, used a tiny bit of a "return false" on the onClick parameter on the <a> tag (a trick I found online), so I could make as though the logo is a link but doesn't direct to anywhere (so when hovering over with the mouse it functions as a link and not as a text)